test/es-module/test-esm-import-flag.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+
+{
+  // known limitation:
+  // imports should not be imported when evaluating text
+  const { status, stderr, stdout } = spawnSync(
+    process.execPath,
+    [
+      '--import',
+      fixtures.path('es-modules', 'mjs-file.mjs'),
+      '-e', 'console.log("log")',
+    ],
+    { encoding: 'utf8' },
+  );
+
+  assert.strictEqual(status, 0);
+  assert.strictEqual(stderr, '');
+  assert.strictEqual(stdout, 'log\n');
+}
+
+{
+  const { status, stderr, stdout } = spawnSync(
+    process.execPath,
+    [
+      '--import',
+      fixtures.path('es-modules', 'mjs-file.mjs'),
```

Comment:
Same everywhere else, IMO paths are suited for this use case.
```suggestion
      fixtures.fileURL('es-modules', 'mjs-file.mjs'),
```